fast64_internal/utility.py

```diff
@@ -1868,3 +1868,22 @@ def upgrade_old_prop(
         print(f"Failed to upgrade {new_prop} from old location {old_loc} with props {old_props}")
         traceback.print_exc()
         return False
+
+
+def create_or_get_world(scene: Scene) -> World:
+    """
+    Given a scene, this function will return:
+    - The world selected in the scene if the scene has a selected world.
+    - The first world in bpy.data.worlds if the current file has a world. (Which it almost always does because of the f3d nodes library)
+    - Create a world named "Fast64" and return it if no world exits.
+    This function does not assign any world to the scene.
+    """
+    if scene.world:
+        return scene.world
+    if bpy.data.worlds:
+        world: World = bpy.data.worlds.values()[0]
+        print(f'No world selected in scene, selected the first one found in this file "{world.name}".')
```

Comment:
actually I just realized this severely spams the console. Could you add a counter for this print and only print it the first 10 times or something
use a dirty global variable :)